app/src/main/java/com/codepath/apps/critterfinder/models/PetModel.java

```diff
@@ -28,6 +30,7 @@
     String contactZip;
     String contactAdress1;
     String contactAdress2;
+    List<String> breeds;   // list of all the breeds the animal may be part of
```

Comment:
any reason not to leverage our existing model for a Breed?

List<Breed> breeds

to maintain our data abstraction? A breed object can be created from a String so we can easily create it from the json response.